src/tools/index.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import logger from "../utils/log";
+import MediaCapabilitiesProber from "./mediaCapabilitiesProber";
+
+logger.setLevel(__LOGGER_LEVEL__);
```

Comment:
:warning: this is the same logger than the RxPlayer. Doing this will also impact it.


Something like:
```js
// a.js
import RxPlayer from "rx-player";

RxPlayer.LogLevel = "DEBUG";
```
```js
// b.js
import "a.js";
import { MediaCapabilitiesProber } from "rx-player/tools";
```

will reset the log level